InfiniTAM/CMakeLists.txt

```diff
@@ -98,11 +98,11 @@ IF(WITH_CUDA)
   target_link_libraries(InfiniTAM ${CUDA_LIBRARIES})
 ENDIF()
 
-add_executable(InfiniTAM_node InfiniTAM_node.cpp)
-target_link_libraries(InfiniTAM_node Engine)
-target_link_libraries(InfiniTAM_node Utils)
+add_executable(infinitam_ros_node infinitam_ros_node.cpp)
```

Comment:
if you use catkin simple, use cs_add_exectuable

remove the `find_package(glog_catkin REQUIRED)` and similar commands... they are invoked by catkin simple if you set them as dependecies.

In general, make sure that the pull-request builds on jenkins.